src/style.css

```diff
@@ -1,3 +1,75 @@
 body {
   margin: 0;
+  font-weight: 500;
+  font-family: Roboto, sans-serif;
+}
+
+:root {
+  --blue-color: #00acdc;
+}
+
+.header {
+  box-sizing: border-box;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  padding: 0 50px;
+  display: flex;
+}
+
+img {
+  margin-top: 1px;
```

Comment:
This margin looks redundant, to remove this blue space you can add font-size: 0 to `logo-link` class
![image](https://github.com/mate-academy/layout_moyo-header/assets/80700247/122a9187-1426-4ce4-a03f-bb0d3ea65be7)